packit/cli/builds/obs_build.py

```diff
@@ -0,0 +1,273 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import os.path
+import xml.etree.ElementTree as ET
+from dataclasses import dataclass
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Optional
+
+import click
+from osc import conf, core
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api, iterate_packages
+from packit.config import get_context_settings, pass_config
+from packit.config.aliases import DEPRECATED_TARGET_MAP
+from packit.config.config import Config
+from packit.config.package_config import PackageConfig
+from packit.constants import (
+    PACKAGE_LONG_OPTION,
+    PACKAGE_OPTION_HELP,
+    PACKAGE_SHORT_OPTION,
+)
+
+logger = logging.getLogger(__name__)
+
+_API_URL = "https://api.opensuse.org"
+
+
+@dataclass(frozen=True)
+class PathEntry:
+    project: str
+    repository: str
```

Comment:
The `PathEntry` is a term used in OBS?

At first, I expected something with file paths..;)

Reply:
This is the representation of a `path` XML element that you can find in the repository configuration, e.g.:
```xml
  <repository name="images">
    <releasetarget project="openSUSE:Factory:ToTest" repository="images" trigger="manual"/>
    <path project="openSUSE:Factory" repository="containerfile"/>
    <path project="openSUSE:Factory" repository="standard"/>
    <arch>local</arch>
    <arch>x86_64</arch>
  </repository>
```

Reply:
Aha, ok. What about adding `Xml` to the name so it's clear?

Reply:
Certainly, I have renamed the class to `XmlPathEntry`.